lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -125,19 +128,19 @@ sub _hmt_csv {
     while (my $row = $csv->getline($fh) or not $csv->eof()) {
         ($row->[23] and $row->[23] eq "Individual") or next;
         my $name = _process_name @{$row}[0 .. 5];
-        
+
         next if $name =~ /^\s*$/;
-        
+
         my $date_of_birth = $row->[7];
         $date_of_birth =~ tr/\//-/;
-        
-        $hmt_ref->{$name}->{dob_epoch} = [];
-        
+
+        $hmt_ref->{$name}->{dob_epoch} = [] unless $hmt_ref->{$name};
```

Comment:
Should this be `$hmt_ref->{$name}->{dob_epoch} = [] unless $hmt_ref->{$name}->{dob_epoch}`?
With current code:
- row 1  "John Smith" has no DOB
- row 2 "John Smith" has DOB
We will not store DOB in this case.

Reply:
Could even be just
```suggestion
        $hmt_ref->{$name}->{dob_epoch} ||= [];
```

`//= []` could be used here provided we `use v5.10;` prior.

Reply:
remove old code, updated as suggested